### PR TITLE
Enable CA1050: Declare types in namespaces

### PR DESCRIPTION
**Pull request by xtqqczze**
https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1050

### CODE REVIEW

File: test/xUnit/Asserts/PriorityAttribute.cs [hunk 1]
```diff
@@ -3,13 +3,16 @@
 
 using System;
 
-[AttributeUsage(AttributeTargets.Method, AllowMultiple = false)]
-public class TestPriorityAttribute : Attribute
+namespace PSTests.Internal
 {
-    public TestPriorityAttribute(int priority)
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = false)]
+    public class PriorityAttribute : Attribute
```
Comment (iSazonov) [inline test/xUnit/Asserts/PriorityAttribute.cs:9]: Why do we rename the attribute? The change is not related to the PR.

Comment (xtqqczze) [reply test/xUnit/Asserts/PriorityAttribute.cs:9]: The `Test` prefix on the attribute appeared unnecessary now it is declared inside the `PSTests` namespace.

**Review by iSazonov — APPROVED** [review]
